README.rst

```diff
@@ -327,28 +355,56 @@ For example, the following will build Kokkos for NVIDIA A100 cards
         -DKokkos_ENABLE_TESTS:BOOL=OFF \
         -DKokkos_ENABLE_LIBDL:BOOL=OFF
     cmake --build build && cmake --install build
-    echo export CMAKE_PREFIX_PATH=/opt/kokkos/4.1.00/AMPERE80:\$CMAKE_PREFIX_PATH
+    export CMAKE_PREFIX_PATH=/opt/kokkos/4.1.00/AMPERE80:\$CMAKE_PREFIX_PATH
```

Comment:
Are we going to specify the kokkos version as `x.y.z` here to make it aligned with line 337? Or can we set it as `4.3.01`?

Reply:
Yes, you are right, I think it will be better to keep the notation `x.y.z`